docs/_docs/templates.md

```diff
@@ -421,56 +421,15 @@ The default is `default`. They are as follows (with what they filter):
 
 ### Includes
 
-If you have small page fragments that you wish to include in multiple places on
-your site, you can use the `include` tag.
+If you have small page fragments that you want to include in multiple places on your site, you can use the `include` tag:
 
 ```liquid
 {% raw %}{% include footer.html %}{% endraw %}
 ```
 
-Jekyll expects all include files to be placed in an `_includes` directory at the
-root of your source directory. This will embed the contents of
-`<source>/_includes/footer.html` into the calling file.
+Jekyll expects all include files to be placed in an `_includes` directory at the root of your source directory. In the above example, this will embed the contents of `_includes/footer.html` into the calling file.
```

Comment:
> so that users don't think we're using "include" as a verb?

yes, that's the intention.. and btw, its not bolding.. but italicizing. Bold is by double asterices.